docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -371,6 +431,7 @@ def plot_knn_multilabels(embeddings, multilabels, samples_idx, n_neighbors=50):
         bars1 = multilabels[idx]
         bars2 = np.mean(multilabels[indices[idx]], axis=0)
 
+        plt.title(filenames[idx])
```

Comment:
Why don't I see the title in the pdf?

Reply:
Ah, the pdf did not yet include this change. Will update it.